web/tailwind.config.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
     },
   },
   plugins: [
+    require('@tailwindcss/line-clamp'),
```

Comment:
Tailwind CSS v3.3 the line-clamp utilities are now included in the framework by default(https://github.com/tailwindlabs/tailwindcss-line-clamp). So we can upgrade tailwindcss to ^3.3.0 and remove this code

Reply:
finished, look again